docs/application/native/guides/machine-learning/machine-learning-service.md

```diff
@@ -23,47 +29,167 @@ Follow the steps below to enable your application to use Machine Learning Servic
    <feature name="http://tizen.org/feature/machine_learning.service">true</feature>
    ```
 
-## An example of usage
+## An example of object detection with Machine Learning Service API
+
+Let some AI developers want to provide a object detection service. The ML service takes an image data from other applications, and give back the image with bounding boxes drawn on it. An example pipeline can be described as this:
+
+```text
+tensor_query_serversrc port=#reserved_appointed_port_number host=localhost connect-type=TCP !
+other/tensors,num_tensors=1,types=uint8,format=static,dimensions=3:320:320:1,framerate=0/1 !
+tensor_transform mode=arithmetic option=typecast:float32,add:0.0,div:255.0 !
+queue ! tensor_filter framework=tensorflow2-lite model=/path/to/yolov5s/model/file.tflite !
+other/tensors,num_tensors=1,types=float32,format=static,dimensions=85:6300:1:1,framerate=0/1 !
+tensor_decoder mode=bounding_boxes option1=yolov5 option2=/path/to/coco/label.txt option4=320:320 option5=320:320 !
+video/x-raw,format=RGBA,width=320,height=320 ! tensor_converter !
+other/tensors,format=static,num_tensors=1,dimensions=4:320:320:1,types=uint8 !
+tensor_query_serversink async=false sync=false
+```
+
+The pipeline expects a 320x320 RGB raw image as input, and pre-process it to invoke the yolov5s tflite model. The inference result is decoded to bounding box image with tensor_decoder. The service provider can upgrade their internal neural network without changing the code of client if input and output layout is preserved.
 
-The following code snippet shows a simple example of the Machine Learning Service API.
+Clients who want to use this service may use their own pipeline communicating with the service, or can use Machine Learning Service API which is more easy to use.
 
-An ML developer adds a pipeline description which includes preprocessing and invoking a neural network model, and an application developer can get the registered pipeline.
+The following code snippet shows an example of how the Machine Learning Service API can be used in this scenario. Note that checking return value and error handling is omitted for brevity.
+
+### AI developers launching the object detection ML service
+
+AI developers can manage the pipeline description.
```

Comment:
```suggestion
AI developers can manage the pipeline description by doing the following:
```
Introduce a list with a complete sentence, not a partial one that's completed by the list items.
Reference: https://developers.google.com/style/lists#introductory-sentences-for-lists